cypress/features/regression/carousel.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Carousel component
+  I want to change Carousel properties
+
+  Background: Open Carousel component page
+    Given I open "Carousel" component page
+
+  @positive
+  Scenario: Initial slide title is Slide 3
+  # When step is run on background as Given
```

Comment:
Explanation should be added also why this line is commented (you could use on of ours :) )

Reply:
Amended comment to what the When step would actually say, as with other component feature files 👍

Reply:
Added "# commented because of BDD default scenario Given - When - Then"